### DISCUSSION TITLE
How to guarantee CSS Load Order?

### DISCUSSION THREAD

**Discussion by dzz007** [nuxt/nuxt#20659] [Questions]
Hi guys, I have roughly the following App.vue: 
```
<template>
...
<Head>
<Link id="theme-link" rel="stylesheet" href="/public.css" />  NOTE: the reason I use Link here is that later in the code I can use id to dynamically change this public.css's location. 
...
</template>
<style src="@/assets/override.scss" />
```

My question is, how can I guarantee override.scss load after public.css? Since override.css is overriding stuff from public.css, order would be important. I have noticed that when SSR=true, the order is expected, but when I set SSR=false, the override.scss actually load before public.css. 

Minimal Repro link: https://stackblitz.com/edit/nuxt-starter-ycw6ts (button is blue when ssr=true, but is red when ssr=false)
Thanks

**Comment by Park-Joyeong** [comment]
Hello.
I'm not familiar english. 
How about this?


`<template>`
`  <Head>`
`    <Link rel="stylesheet" href="/public.css" />`
`  </Head>`
`  <div>`
`    <button class="buttoncls">abc</button>`
`  </div>`
`</template>`

`<script>`
`  import '@/assets/override.css'`
`</script>`
